### PR TITLE
Redesign Student Expert Volunteer Section

### LINKED ISSUE #752: Redesign Student,Expert,Volunteer Section
**Is your feature request related to a problem?**
As discussed we need to redesign the entire Student, Expert, Volunteer section.
https://sef.discourse.group/t/redesigning-the-students-experts-volunteers-section/55/76?u=gimhan_minion

**Describe the solution you'd like**
Redesign according to the design provided by the design team.

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose
The purpose of this PR is to fix #752 

## Goals
To redesign Student Expert Volunteer Section according to the design

## Approach
1. Deleted the currently existing section
2. Created a new section according to the design
 
### Preview Link

https://pr-753-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by akshika47** [comment]
There is an extra "Lorem ipsum dolar sit amet"

You also need to hyperlink the projects to their respective pages (For example OneLive, ScholarX etc)

We might need to connect few more call to action links.

**Comment by akshika47** [comment]
@Gimhan-minion can I bother you to put a Heading for the time being? I asked you to remove the Lorem Epsum but did not ask you to put a heading there. Sorry for that.

Lets put this as the heading: At SEF, we have something in store for everyone!

We would need some colour alternation for the background or some dividers, but we can do them in a separate PR! For this let's add that heading and we can finalise this PR!

**Comment by YohanAvishke** [comment]
Nice job @Gimhan-minion 😍 Few suggestions for the mobile view.

- Increase the bottom margin of `At SEF, we have something ...`
- Move the images under the titles.
- Center the images with the titles.

Example:
image

WDYT?

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -203,77 +203,52 @@ <h5 class="display-3 text-center project-stat">
     <section class="section">
         <div class="container my-5">
             <div class="col">
-                <h4 class="font-weight-900 text-dark text-center">You could be a student, an expert or a volunteer. At SEF,
-                    we have something in store for everyone!</h4>
+                <h4 class="font-weight-900 text-center mb-md-5"> At SEF, we have something in store for everyone!</h4>
```
Comment (anjula-sack) [inline src/main/webapp/index.html]: The title should be bigger and do something better to make it stand out than the other titles like student, expert

Comment (anjula-sack) [inline src/main/webapp/index.html]: Reduce the space little-bit between the main title and "our projects" section

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Thanks @Gimhan-minion! small change is required

File: src/main/webapp/index.html (hunk 1 above)
Comment (Gimhan-minion) [reply src/main/webapp/index.html]: Done

Comment (Gimhan-minion) [reply src/main/webapp/index.html]: Done

**Review by anjula-sack — CHANGES_REQUESTED** [review]
There's a bug in the mobile view.
image

**Review by anjula-sack — APPROVED** [review]
Thank you! @Gimhan-minion

**Review by Piumal1999 — APPROVED** [review]
Well done @Gimhan-minion Thank you!!

**Pull request merged**